projects/etcd/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```

Comment:
The etcd Authors

Reply:
The files in this repo [must have the OSS-fuzz copyright header](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#copyright-headers), however, once etcd has been integrated we can move both the fuzzer and build script upstream with amended copyright headers.

Reply:
I'm sorry. Started to read this with etcd-repo mindset.